crates/noirc_evaluator/src/ssa/opt/die.rs

```diff
@@ -210,13 +210,13 @@ mod test {
         //     v7 = load v6
         //     v8 = add v7, Field 1
         //     v9 = add v7, Field 2
-        //     call println(v8)
+        //     call to_le_bits(v8)
         //     return v9
         // }
         let ssa = ssa.dead_instruction_elimination();
         let main = ssa.main();
 
         assert_eq!(main.dfg[main.entry_block()].instructions().len(), 1);
-        assert_eq!(main.dfg[b1].instructions().len(), 6);
+        assert_eq!(main.dfg[b1].instructions().len(), 4);
```

Comment:
Why is this `4` instructions now? The expected output comment above shows the same number of instructions?

Reply:
This due to dead instruction elimination. `println` has sideeffects whereas `toBits` doesn't so it gets eliminated.

Reply:
Because `println` was swapped to `to_le_bits` which has no side effects and so was removed by DIE, along with its argument `v8` which was unused elsewhere.

Can we switch to `AssertConstant` instead? That way we can keep the same instruction count. I'm also wary of calling `to_le_bits` but not having the call instruction have the return values of that function. Although it does not matter to DIE, I think the SSA we generate for tests should generally be in a correct form.